README.md

```diff
@@ -392,6 +392,7 @@ The following bundle settings are provided under the `BundleKeys` object:
 
 Name                  | Description
 ----------------------|-------------
+annotations           | An optional HOCON string representing additional metadata that you may wish to associate with a bundle. Key names should be in accordance with the [OCI image annotation](https://github.com/opencontainers/image-spec/blob/master/annotations.md) conventions.
```

Comment:
I'd add an annotations example, e.g. 

```
{
  reference-id = "124f534as",
  zones = ["us-east", "us-west"]
}
```

Users might not familiar with the HOCON syntax, especially that they need to wrap the key value pairs in `{}`.